### ISSUE TITLE
Start Mailman keeps Loading...

### ISSUE THREAD

**Issue by espirit** [brentadamson/mailman#17]
When activating Mailman, it pops up a dialog with "Loading...".
This never stops. 

I switched to the Chrome browser, since I had seen a similar issue with YAMM, but the problem is on Chrome as well.

**Comment by brentadamson** [comment]
If you open up a browser console is there an error message?

**Comment by espirit** [comment]
SOLVED.

I had multiple accounts open in the same Browser.
I guess it used a different account instead the one of the sheet document.

**Comment by brentadamson** [comment]
Yeah, Sheets isn't very good at logging you out completely. I imagine it happens a lot to people that switch a lot between a work and personal email address. Is there a way we can detect it and handle it better? A continuous loading dialog isn't very helpful.
